department-of-veterans-affairs/caseflow
# Calvin/appeals 13229

Resolves [APPEALS-13229](https://vajira.max.gov/browse/APPEALS-13229)

### Description
Added static veteran ID's to cavc_ama_appeals.rb to align with other cavc remand seeding files such as priority_distribution.rb and static_test_data.rb

### Acceptance Criteria

- [ ] For each environment instructions on how to find and/or create seed data is added to https://github.com/department-of-veterans-affairs/caseflow/wiki/CAVC-Remands

- [ ] For demo https://github.com/department-of-veterans-affairs/caseflow/tree/master/db/seeds is reviewed to see if CAVC Remands are already being created

   1. If so they are given static veteran IDs so it can be more easily found on redeploys to demo

   2. If none are found a new "Static CAVC Remand" seed file is created

### Testing Plan
1. Go to terminal to rerun the seed file for cavc_ama_appeals.rb
   1. Open a rails console (make c)

2. Run the following commands in the rails console:
   1. `RequestStore[:current_user] = User.system_user`
   2. `Dir[Rails.root.join("db/seeds/*.rb")].sort.each { |f| require f }`
        * If error occurs, press Q to continue.

3. For the desired seed file, run the following command:
   1. `Seeds::CavcAmaAppeals.new.seed!`

4. Ensure seed file is successfully run through rails console
